### DISCUSSION TITLE
cURL update to 8.2.0

### DISCUSSION THREAD

**Discussion by knthalva** [curl/curl#11724] [Q&A] [answered]
Hi Everyone, need help on how to update the curl.exe in windows into upper version. thank you.

**Comment by bagder** [comment, marked as answer]
The latest curl version is right now **8.2.1**

You can always download the latest curl release for Windows from here: https://curl.se/windows/

That is a *separate* and  *different* build from the one that is bundled and shipped as a component of Windows itself. If you have an issue with the version of curl that is shipped in Windows, you need to take that discussion to your Microsoft representative. It is **strongly** advised that you do not mock around with the system installed curl tool binary.

**Reply by knthalva** [reply]
can I just replaced the curl.exe in the sys32?

**Reply by bagder** [reply]
You *can*, but you will regret it. For example that action will prevent your "windows upgrade" from working.

Again: It is **strongly** advised that you do not mock around with the system installed curl tool binary

**Reply by jzakrzewski** [reply]
Obligatory link with some more information ;) : https://daniel.haxx.se/blog/2023/04/24/deleting-system32curl-exe/

**Reply by knthalva** [reply]
Thank you for your help.
